src/ReactiveDomain/ReactiveDomain/Transport/TcpInboundCommandHandler.cs

```diff
@@ -1,85 +0,0 @@
-﻿using System;
-using System.Collections.Concurrent;
-using System.Collections.Generic;
-using System.Configuration;
-using System.Diagnostics;
-using System.Net;
-using System.Net.NetworkInformation;
-using System.Net.Sockets;
-using System.Threading;
-using ReactiveDomain.Transport.CommandSocket;
-using ReactiveDomain.Transport.Framing;
-using ReactiveDomain.Bus;
-using NLog;
-using ReactiveDomain.Messaging;
-
-namespace ReactiveDomain.Transport
-{
-    /// <summary>
-    /// Commands that come off of the TCP/IP connection go through the inboundCommandQueuedHandler,
-    /// which then hands them to this class via Handle(message).  This class casts them to
-    /// Commands and then fires them on the MessageBus.  When Fire() returns, this class
-    /// takes the CommandResponse and publishes it (cast as a Message) on the outboundMessageQueuedHandler.
-    /// </summary>
-    public class TcpInboundCommandHandler : IHandle<Message>
```

Comment:
I think this was moved, or renamed, not dropped. let me check

Reply:
OK. Can we comment out/disable the files instead of deleting them outright, so we at least have something to refer to when re-implementing? This is going to be critical to fix, as we have a time-critical project planned for early March that will need these remote buses.